Lib/test/test_warnings/__init__.py

```diff
@@ -794,6 +794,33 @@ def test_stderr_none(self):
         self.assertNotIn(b'Warning!', stderr)
         self.assertNotIn(b'Error', stderr)
 
+    def test_warn_explicit(self):
```

Comment:
This name is too general, but the test isn't a general test of `warn_explicit()`. Use something more specific like test_issue31285.